test/libsolidity/smtCheckerTests/operators/ternary_operator_external_code_1.sol

```diff
@@ -0,0 +1,32 @@
+abstract contract D {
+	function d() external virtual returns (uint);
+}
+
+contract C {
+
+	D d;
+	uint v;
+	bool guard = true;
+
+	function inc() public {
+		++v;
```

Comment:
Why can't this overflow?

Reply:
I think it can, but I specifically wanted to test the underflow here, so the test specifies only `underflow` target.

Reply:
ah true, ok